### PR TITLE
Updated hyperlink to fivetran_log__credit_mar_destination_history model

### PR DESCRIPTION
**Pull request by ferrap**
The current hyperlink redirects to an empty document for fivetran_log__credit_mar_destination_history model

Pull Request
**Are you a current Fivetran customer?** 
Fivetran employee

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Refer to the table below for a detailed view of all models materialized by defau
...
 | [fivetran_log__mar_table_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__mar_table_history)     | Each record represents a table's active volume for a month, complete with data about its connector and destination.                             |
-| [fivetran_log__credit_mar_destination_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__credit_mar_destination_history)    | Each record represents a destination's consumption by showing its MAR, total credits used, and credits per millions MAR.                             |
+| [fivetran_log__credit_mar_destination_history](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_log__usage_mar_destination_history)    | Each record represents a destination's consumption by showing its MAR, total credits used, and credits per millions MAR.                             |
```
Comment (fivetran-joemarkiewicz) [inline README.md:31]: Thanks so much for opening this PR @ferrap! You are correct that this should be updated, thanks for applying the update to the link 🙌 

Additionally, we should probably update the name of the model and the description to be consistent with the information [here](https://github.com/fivetran/dbt_fivetran_log/blob/9f89183075bf03938f81044b673cfb231e353443/models/fivetran_log.yml#L113-L117). Would you be open to applying those updates? Once you do, I can approve and merge!

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
Thanks so much for opening this PR @ferrap 🚀 

I agree with your requested change, but have one small request to update before this will look good to merge. Let me know if you have any questions!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Brilliant, thanks so much @ferrap! 🎉 

LGTM

**Comment by ferrap** [comment]
I noticed the buildkite job has failed and changes are not reflected in the documentation. Do you have permission to re-launch the job? @fivetran-joemarkiewicz

**Comment by fivetran-joemarkiewicz** [comment]
I do! BuildKite is not yet rolled out into the `main` branch (which is why it failed). the contents of PR #64 will include the BuildKite migration. Until then the job will show as failed.

The documentation on the dbt hub will only show following the official release (which we are waiting to cut until we can merge PR #64). The docs on Fivetran and the GitHub repo should be up to date following your most recent PR merge.

**Pull request merged**